Public/Src/Engine/Dll/Distribution/Grpc/GrpcConstants.cs

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace BuildXL.Engine.Distribution.Grpc
+{
+    internal static class GrpcConstants

```

Comment:
Did you consider adding the const timeouts in the ConnectionManger here as well?